char_index.py

```diff
@@ -0,0 +1,29 @@
+char_position = 0
+lowecase_chars = ['countat1','a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z']
+return_message = "Position of alphabet: {char_position}"
+character_argument_max_len = 1
+    
+def position(character_argument: str) -> str:
+    """with a letter, return its position in the alphabet
+    
+    Args:
+    String, a single character 
+    
+    Returns
+    String, a string in the format of 'Position of alphabet: char_position'
+    
+    Raises
+    ca
+    """
+    
+    # Sanity Checks
+    if not (isinstance(character_argument, str) and len(character_argument) == character_argument_max_len):
+        raise ValueError("You're SOOO stoooodpid.")
+        
+    if character_argument not in lowecase_chars:
+        raise ValueError("YOU GET NOTHING")
```

Comment:
just a note strings have a `.islower()` method which returns true if all the characters are lowercase so you could just say: 
```suggestion
    if not character_argument.islower():
        raise ValueError("YOU GET NOTHING")
```

Reply:
that's smarter.

Reply:
I have been writing python professionally for over 4 years.